frontend/src/index.tsx

```diff
@@ -1,13 +1,24 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import React, { Suspense } from "react";
+import ReactDOM from "react-dom";
+import { BrowserRouter, Route, Routes } from "react-router-dom";
+import App from "./App";
+import AppTitle from "./Components/AppTitle/AppTitle";
+import GlobalStyles from "./Globalstyles";
+import reportWebVitals from "./reportWebVitals";
 
 ReactDOM.render(
-    <React.StrictMode>
-        <App />
-    </React.StrictMode>,
-  document.getElementById('root')
+  <React.StrictMode>
+    <Suspense fallback="Loading...">
+      <GlobalStyles />
+      <BrowserRouter>
+        <Routes>
+          <Route path="/" element={<AppTitle children={"ALEX"}/>} />
+          <Route path="/home" element={<App />} />
+        </Routes>
+      </BrowserRouter>
+    </Suspense>
```

Comment:
```suggestion
      <GlobalStyles />
      <BrowserRouter>
        <Routes>
          <Route path="/" element={<AppTitle children={"ALEX"}/>} />
          <Route path="/home" element={<App />} />
        </Routes>
      </BrowserRouter>
```
Since you don't use I18n you don't need the suspense tag

Reply:
i thought for later... but you're right, i'll take it out